### DISCUSSION TITLE
안녕하세요 작가님 질문이 있습니다.

### DISCUSSION THREAD

**Discussion by Daehyeon2yam** [fsoftwareengineer/todo-application#54] [질문과 답변] [answered]
현재 실습코드 3-24까지 따라하면서 프론트 엔드 동작 방식에 대해 학습하고 있습니다.

material ui를 적용 전에는 check box 등이 잘 나타났는데 

material ui를 적용하면서 실습을 진행하면서는 
<img width="1391" alt="스크린샷 2022-01-10 오후 5 27 35" src="https://user-images.githubusercontent.com/68681500/148736638-5baa8fdd-1eee-445b-a905-6108eab4e5b1.png">

localhost:3000에 아무런 화면이 나타나지 않는 문제가 발생하는데

어떤식으로 해결을 해야할까요??

**Comment by Daehyeon2yam** [comment]
혹시몰라서 작가님의 git에 3-2 부분의 코드를 복사해서 붙여넣고 실행을 해봤는데도 동일한 ui가나와서 질문드립니다.

**Reply by fsoftwareengineer** [reply]
아래의 항목을 본문에 추가 해 주시기 바랍니다.

- 브라우저 console 창에 에러가 있는지, 에러가 무엇인지.
- 브라우저 디버깅 창의 dom트리.
- 에러가 없다면 material ui의 일부 코드, 예를 들어 List하나만 출력 한 결과.

**Comment by Daehyeon2yam** [comment]
지금 확인하고 메일로 전달해드립니다.
 
 
 
-----Original Message-----

**Comment by Daehyeon2yam** [comment]
<img width="1026" alt="스크린샷 2022-01-18 오후 4 08 02" src="https://user-images.githubusercontent.com/68681500/149889515-82e81bb9-cbfd-4c8a-b8d8-d818328f166a.png">

콘솔의 오류 메시지 로그입니다.

**Reply by fsoftwareengineer** [reply]
리액트 버전을 올리거나 노드js를 재설치 해봐야 할 것 같습니다.

**Reply by fsoftwareengineer** [reply]
node_modules 폴더랑 package.json.lock 삭제후 npm install을 해도 결과가 같나요 ?

**Comment by Daehyeon2yam** [comment]
리액트 버전은 17.0.1 이며 노드는 다운그래이드해서 16.13.2 버전 재설치 및 말씀해주신 
 
모듈이랑 pacage.json-lock을 삭제후 다시 npm start를 해보았으나 이전과 상황이 동일합니다. 
 
 
-----Original Message-----

**Comment by fsoftwareengineer** [comment]
`npm ls react-dom` 실행 결과를 확인 해 주시기 바랍니다. React-dom 버전을 낮춰서 진행 해 보는 것도 한 방법일 것 같습니다. 책에서 사용하는 버전은 아래와 같습니다. 

```
    "react-router-dom": "^5.2.0",
```

**Comment by Daehyeon2yam** [comment, marked as answer]
작가님 감사합니다 덕분에 문제해결 됐네요
 
새로 폴더만들어서 npm 초기화 및 react dom 버전 낮춰서 실행해보니 정상 작동합니다 감사합니다!
-----Original Message-----
